src/app/selectors.ts

```diff
@@ -49,8 +49,6 @@ export const fiatConnectCashInEnabledSelector = (state: RootState) =>
 export const fiatConnectCashOutEnabledSelector = (state: RootState) =>
   state.app.fiatConnectCashOutEnabled
 
-export const coinbasePayEnabledSelector = (state: RootState) => state.app.coinbasePayEnabled
```

Comment:
could also clean the `coinbasePayEnabled` redux state and remote config setup, but there are other remote config flags we can cleanup, which all need migrations, so can do that in a single PR